addons/website_crm/controllers/main.py

```diff
@@ -8,6 +8,26 @@
 
 class WebsiteForm(main.WebsiteForm):
 
+    #  Prefill the fields of contactus form with logged in partner information. They are put in request.params and
+    #  Those are used as default attribute values in the corresponding template.
+    @http.route('/contactus', type='http', auth="public", method=['GET'], website=True, sitemap=False, csfr=False)
```

Comment:
Issue I see here is that contactus page could be on another route (it is configurable). You may rename it to /info for example. Here hardcoding it breaks that logic. Maybe @JKE-be could help as I don't know the internals of the website forum. It seems it support request.params (as you did here), but not sure how to set them in a clean way without hardcoding contactus route.

Reply:
Thanks,

Contactus should not be a controller, but keep as a page.
Allowing to rename url, be optional on your website, ... 

Sitemap=False ?
method=GET for a form ?

Reply:
Maybe, you could add default value with data-for attribute directly in form instead to have a controller for it ?